incuna-sass/mixins/_font-face.sass

```diff
@@ -26,40 +29,24 @@
 //     @include font-face("roboto-medium", "Roboto-Medium", "roboto", "sans-serif")
 //     @include font-face("roboto-bold-italic", "Roboto-Bold-Italic", "roboto", "sans-serif")
 
-// You can then use the font mixin to apply these fonts to your elements:
-//     @include font(roboto-regular)
+// You can then apply this font using font-family: "font-name"
+// You can set the font weight or style (assuming you've added the
+// relevant files in a font-face mixin) using font-weight and font-style
+// as you would normally
 
-// I recommend setting font names as variables in your projects locally
-// to make it easier to set fonts, eg:
+// IMPORTANT:
+// If you have several weights/styles of the same font make sure
+// you give them the same font name so the weight / style can be set correctly
+// You can only apply font weights and styles that you have addedfiles for.
+// Setting the weight or style to something you have not provided a
+// font file for will result in faux browser styles
 
-// $font-main: font-name
-// .main
-//     @include font($font-main)
-
-// for different font-weights and styles:
-
-// $font-bold: font-name-bold
-// $font-medium-italic: font-name-medium-italic
-// .main
-//     h2
-//         @include font($font-bold)
-//     em
-//         @include font($font-medium-italic)
-
-// or for an accent font:
-
-// $font-fancy-script: fancy-script-font-name
-// .fancy-element
-//     @include font($font-fancy-script, cursive)
-
-// $font-formal-serif: formal-serif-font-name
-// .formal-element
-//     @include font($font-formal-serif, serif)
-
-@mixin font-face($style-name, $file, $family, $category: '', $woff2: false, $IE-8: false)
+@mixin font-face($font-family, $weight, $style, $file, $family, $category: '', $woff2: false, $IE-8: false)
     $filepath: $family + '/' + $file
```

Comment:
wouldn't weight be needed in here to select eg `lato-bold.otf` as opposed to `lato.otf`?